crates/daphne/src/protocol/aggregator.rs

```diff
@@ -383,7 +383,7 @@ pub(crate) enum ReportProcessedStatus {
     Rejected(TransitionFailure),
 }
 
-#[derive(Default, Debug)]
+#[derive(Default, Debug, Clone, Copy)]
 pub enum ReplayProtection {
     #[default]
     Enabled,
```

Comment:
Change the variant "Disabled" to "InsecureDisabled". TBH it would be nice if we could put this whole enum behind "test-utils".

Reply:
We can't put this enum behind test utils for as long as we have an override to disabled replay protection in production. After that's gone we can do this